diff --git a/src/kerberos.c b/src/kerberos.c
index f73e65a..1ba32dc 100644
--- a/src/kerberos.c
+++ b/src/kerberos.c
@@ -16,7 +16,7 @@
  * DRI: Cyrus Daboo, cdaboo@apple.com
  **/
 
-#include <Python/Python.h>
+#include "Python.h"
 
 #include "kerberosbasic.h"
 #include "kerberosgss.h"
diff --git a/src/kerberosbasic.c b/src/kerberosbasic.c
index 4135694..0fe3a3a 100644
--- a/src/kerberosbasic.c
+++ b/src/kerberosbasic.c
@@ -16,7 +16,7 @@
  * DRI: Cyrus Daboo, cdaboo@apple.com
  **/
 
-#include <Python/Python.h>
+#include "Python.h"
 #include "kerberosbasic.h"
 
 #include <stdio.h>
diff --git a/src/kerberosgss.c b/src/kerberosgss.c
index 464c066..d1e0167 100644
--- a/src/kerberosgss.c
+++ b/src/kerberosgss.c
@@ -16,7 +16,7 @@
  * DRI: Cyrus Daboo, cdaboo@apple.com
  **/
 
-#include <Python/Python.h>
+#include "Python.h"
 #include "kerberosgss.h"
 
 #include "base64.h"
